web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1785,17 +1822,15 @@
                       version: '1.0.0'}));
               }
 
-              //Create layer
+              // Create layer
               var olLayer = new ol.layer.Tile({
-                extent: map.getView().getProjection().getExtent(),
                 name: getCapLayer.Identifier,
                 title: getCapLayer.Title,
                 label: getCapLayer.Title,
                 source: new ol.source.WMTS(options),
                 url: url,
-                urlCap: urlCap,
-                cextent: gnOwsCapabilities.getLayerExtentFromGetCap(map,
-                    getCapLayer)
```

Comment:
Calling the `getLayerExtentFromGetCap()` function always returns `null` for WMTS layers from what was observed, so it makes no sense to call it at all.